platform/admin/core/admin-core-logviewer/src/main/webapp/js/components/log-entry.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+var React = require('react')
+var moment = require('moment')
+
+var levels = require('../levels')
+
+var format = function (time) {
+  return moment(time).format('D MMM YYYY, HH:mm:ss')
+}
+
+var styles = function (level) {
```

Comment:
I am not sure how good of an idea this is.  It seems like there are a lot of drawbacks that need to be considered before we adopt using inline styles.

https://byjoeybaker.com/react-inline-styles
http://jamesknelson.com/why-you-shouldnt-style-with-javascript/
https://css-tricks.com/the-debate-around-do-we-even-need-css-anymore/

@codice/ui